scania-digital-design-system/tegel
# docs(popover): angular workarounds

**Describe pull-request**  
This PR documents the Angular workaround for using the popover-menu/canvas in an angular context. I know this workaround isn't needed for vanilla components in Angular. But I think it's better to not confuse people and just recommend they always do it this way instead.


**Solving issue**  
Fixes: [CDEP-2677](https://tegel.atlassian.net/browse/CDEP-2677)

**How to test**  
1. Go to Storybook
2. Check in Popover Canvas/Menu -> Notes
